packages/flutter/lib/src/services/text_formatter.dart

```diff
@@ -319,8 +319,8 @@ class LengthLimitingTextInputFormatter extends TextInputFormatter {
   /// The limit on the number of characters (i.e. Unicode scalar values) this formatter
   /// will allow.
   ///
-  /// The value must be null or greater than zero. If it is null, then no limit
-  /// is enforced.
+  /// The value must be null or greater than zero. If it is null or -1, then no
```

Comment:
Also @justinmc it appears the docs here talk in length about how we are counting characters in Unicode scalar values. I believe that's no longer true?

Reply:
Ah good catch.  Yeah below it incorrectly says it *doesn't* count grapheme clusters correctly.

Reply:
I'll update that in a separate PR right now.

Reply:
That separate PR is here for the record: https://github.com/flutter/flutter/pull/66582